.github/workflows/workflow.yml

```diff
@@ -0,0 +1,63 @@
+name: mate-deployment
+run-name: ${{ github.actor }} - ${{ github.sha }}
+
+on:
+  push:
+    branches:
+     - main
+     - develop
+  pull_request:
+    branches:
+     - main
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: 3.8
+
+jobs:
+  python-ci:
+    runs-on: ubuntu-latest
+
+    defaults:
+     run:
+      working-directory: ./src
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        name: Setup Python ${{ env.PYTHON_VERSION }}
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install dependencies
+        run: |
+           python -m pip install --upgrade pip
+           pip install flake8 coverage
+           pip install -r requirements.txt
+
+      - name: Lint with flake8
+        continue-on-error: true
+        run: |
+          cd ../
```

Comment:
What is the purpose of this step?
If it is needed you could use `working-directory` config.
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#defaultsrunworking-directory